docs/4.2/pages/architecture/teleport-architecture-overview.mdx

```diff
@@ -1,12 +1,10 @@
 ---
-title: Teleport Architecture Overview
-description: Basic concepts and architecture of Teleport. What is an SSH cluster? How certificate-based SSH authentication works? How SSH audit works?
```

Comment:
Wow. They shouldn't have been. My only idea is that is a side effect of rebasing to master before posting this PR.

Reply:
Ok, I checked it once more and I think that the problem is with diff display. It tries to compare 4.4 version with 4.2 version because they have mostly the same content and 4.2. didn't have description yet. But I checked 4..4 version and it is still in place. You can compare it bu commits, I tried to make them manageable and only changing one thing at the time, their diff is better.